src/app/components/core/configuration-manager/configuration-manager.component.ts

```diff
@@ -257,14 +182,15 @@ export class ConfigurationManagerComponent implements OnInit {
         this.validConfigForm = false;
         this.alertService.success('Configuration updated successfully.', true);
         this.ngProgress.done();
+        this.selectedNode = {id: categoryName, description: categoryDescription};
```

Comment:
what if  categoryDescription is missing?

Reply:
`categoryDescription` will always present there.